test/gomodgraph.test.ts

```diff
@@ -0,0 +1,13 @@
+import * as fs from 'fs';
+import { parseGoModGraph, } from '../lib';
+import { createFromJSON } from '@snyk/dep-graph';
+
+const load = (filename: string) =>
+    fs.readFileSync(`${__dirname}/fixtures/${filename}`, 'utf8');
```

Comment:
I would recommend `fs.readFileSync(path.resolve(__dirname, 'fixtures',  filename), 'utf8');`
It allows the support for different OS e.g. windows. 
I know that we are not supporting it yet in this repo suite of tests but worth think about it.

n.b. is just a suggestion ^^